### PR TITLE
Redo search in map

### PR DESCRIPTION
**Pull request by LinX64**

### CODE REVIEW

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt [hunk 1]
```diff
@@ -158,6 +154,14 @@ class SearchFixedResultActivity : AppCompatActivity(), OnMapReadyCallback,
         })
     }
 
+    private fun setEditTextWithStyle(address: String, etPlace: EditText?) {
+        etPlace?.apply {
+            this.hint = address
+            this.textSize = 15.0f
+            this.setHintTextColor(ContextCompat.getColor(this.context, R.color.black_color))
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt:161]: I am not sure, but can we not set this style in the xml?

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt [hunk 2]
```diff
@@ -146,7 +142,7 @@ class SearchFixedResultActivity : AppCompatActivity(), OnMapReadyCallback,
                 val lat = place.latLng?.latitude
                 val lng = place.latLng?.longitude
 
-                etPlace?.hint = address
+                setEditTextWithStyle(address, etPlace)
                 if (lat != null && lng != null) {
                     moveMapToSelectedLocationAndRefresh(lat, lng)
                 }
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt:148]: I believe the redoButton.gone() should be here, after calling moveMapToSelectedLocationAndRefresh()

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt (hunk 1 above)
Comment (LinX64) [reply app/src/main/java/pl/llp/aircasting/ui/view/screens/search/SearchFixedResultActivity.kt:161]: No, we can't!

**Comment by LinX64** [comment]
I've merged this PR with [this](https://github.com/HabitatMap/AircastingAndroid/pull/494) - everything related to UI improvements goes there.

**Pull request closed without merging**